change/@fluentui-react-charting-effaa597-2236-4e8c-b8f3-91a2422c9f44.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Add a paragraph in PR description about the issue, what was fixed and the 2 flags that have been used to limit the rollout.

Reply:
Done